packages/data-stores/src/wpcom-plans-ui/reducer.ts

```diff
@@ -14,8 +15,19 @@ const showDomainUpsellDialog: Reducer< boolean | undefined, WpcomPlansUIAction >
 	return state;
 };
 
+const selectedStorageAddOnsForPlans: Reducer<
```

Comment:
**Note:**

Naming this slice of state felt odd because I wasn't sure how to interpret the existing convention where a sibling reducer is called `showDomainUpsellDialog`. In the end, I just tried to make it as specific as possible, but I'm totally open to changing this